src/java.desktop/share/classes/javax/swing/plaf/metal/MetalHighContrastTheme.java

```diff
@@ -51,8 +51,6 @@
                               204, 204, 204);
     private static final ColorUIResource secondary3 = new ColorUIResource(
                               255, 255, 255);
-    private static final ColorUIResource controlHighlight = new
-                              ColorUIResource(102, 102, 102);
```

Comment:
I didn't check it on Windows(AFAICS it is used for Windows only), but I've done a quick check how it looks on Linux with both colors.

Using `controlHighlight`  in `getControlHighlight()`:
![using controlHighlight](https://user-images.githubusercontent.com/77687766/112357602-bf00dd80-8ce0-11eb-9a8d-0faeb6d1940b.png)

Using `secondary2`  in `getControlHighlight()`:
![using secondary2](https://user-images.githubusercontent.com/77687766/112357736-db9d1580-8ce0-11eb-8e1a-527ec327874f.png)

And it seems that the option with unused `controlHighlight` color is more fits the definition of high contrast.

Have you checked why its usage was removed?

Reply:
As far I see in the history of this file, it was `secondary2` that was used from beginning, so I kept it and removed the unused.